### PR TITLE
New payment receipt email

### PR DESCRIPTION
**Pull request by kberzinch**
Replaces the existing payment receipt email with a plain-text version.

### CODE REVIEW

File: database/factories/TravelFactory.php [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    /**
+     * Define the model's default state.
+     *
+     * @return array<string,string|int|\DateTime>
+     */
+    public function definition()
+    {
+        return [
+            'name' => $this->faker->word(),
+            'primary_contact_user_id' => User::all()->random()->id,
```
Comment (evan10s) [inline database/factories/TravelFactory.php:26]: I'd prefer something that creates a user if one doesn't exist. Or does this do that?

File: database/factories/TravelFactory.php [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+            'destination' => $this->faker->word(),
+            'departure_date' => $this->faker->dateTimeBetween('now', '1 year'),
+            'return_date' => $this->faker->dateTimeBetween('now', '1 year'),
```
Comment (evan10s) [inline database/factories/TravelFactory.php:29]: There's no guarantee here that return date is after departure date.  Maybe adjust the time ranges for departure/return date so they can't overlap the wrong way.

File: resources/views/mail/paymentreceipt.blade.php [hunk 3]
```diff
@@ -0,0 +1,11 @@
+Hi {{ $payment->payable->user->preferred_first_name }},
+
+@if($payment->method === "square")
+We received your online payment of ${{ number_format($payment->amount, 2) }} for {{ $payable_name }}.
+
+You can view more details about your payment at {{ $payment->receipt_url }}
```
Comment (evan10s) [inline resources/views/mail/paymentreceipt.blade.php:6]:
```suggestion
You can view more details about your payment at {{ $payment->receipt_url }}.
```

**Review by JoshOldenburg — APPROVED** [review]

File: database/factories/TravelFactory.php (hunk 1 above)
Comment (kberzinch) [reply database/factories/TravelFactory.php:26]: This picks a random user. That's what the `DuesTransactionFactory` was already doing so I copied that.

I think this is reasonable default behavior; if something more specific is required for a test then the attribute can be overriden - https://laravel.com/docs/9.x/database-testing#overriding-attributes

File: resources/views/mail/paymentreceipt.blade.php (hunk 3 above)
Comment (kberzinch) [reply resources/views/mail/paymentreceipt.blade.php:6]: Will need to test how that renders in real clients, but ok

**Review by evan10s — APPROVED** [review]

**Review by JoshOldenburg — APPROVED** [review]

**Pull request merged**